practice/tic-tac-toe/.cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Looks like your .gitignore file in your main `codeguild` dir is not working. It should contain `__pycache__` and `.cache`. Please ensue it is up to date and do commit the `.gitignore` file itself. It should look like [my example one](https://github.com/PdxCodeGuild/Full-Stack-Day-Class/blob/master/demos/example_gitignore).